File: coreui/src/main/java/com/mbg/module/ui/activity/BaseActivity.java
package com.mbg.module.ui.activity;




import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.jaeger.library.StatusBarUtil;
import com.mbg.module.common.util.LocaleUtils;

public abstract class BaseActivity extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        LocaleUtils.setLocale(this);
    }

    public void setStatusBarColor(int color, int statusBarAlpha){
        StatusBarUtil.setColor(this,color,statusBarAlpha);
    }

}

File: coreui/src/main/java/com/mbg/module/ui/kotlin/activity/BaseActivity.kt
package com.mbg.module.ui.kotlin.activity

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.jaeger.library.StatusBarUtil
import com.mbg.module.common.util.LocaleUtils

abstract class BaseActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        LocaleUtils.setLocale(this)
    }

    fun setStatusBarColor(color: Int, statusBarAlpha: Int) {
        StatusBarUtil.setColor(this, color, statusBarAlpha)
    }
}